src/bci_build/package/openjdk.py

```diff
@@ -44,6 +46,16 @@ def _get_openjdk_kwargs(
             # prevent ftbfs on workers with a root partition with 4GB
             "_constraints": generate_disk_size_constraints(6)
         },
+        "replacements_via_service": [
+            Replacement(
+                regex_in_build_description="%%java_version%%",
+                package_name=(
+                    f"java-{java_version}-openjdk-devel"
+                    if devel
+                    else f"java-{java_version}-openjdk"
```

Comment:
Isn't `f"java-{java_version}-openjdk"` not in both the devel & non-devel image?

Reply:
it is, but do we want to rely on that fact?